examples/vdpa/main.c

```diff
@@ -179,6 +180,19 @@ start_vdpa(struct vdpa_port *vport)
 			"register driver failed: %s\n",
 			socket_path);
 
+	/*vdpa device should get feature from device and not use builtin_net_driver*/
```

Comment:
white space before and after /* and */ is needed.

you need to review those cosmetics once before posting here.